setup.py

```diff
@@ -100,19 +91,11 @@ def emit(self, record):
             _unittests_only(suite, mod.test_suite())
     return suite
 
-doclines = __doc__.split("\n")
-
-def read_file(*path):
-    base_dir = os.path.dirname(__file__)
-    file_path = (base_dir, ) + tuple(path)
-    with open(os.path.join(*file_path), 'rb') as file:
-        return file.read()
-
-long_description = str(
-    ("\n".join(doclines[2:]) + "\n\n" +
-     ".. contents::\n\n" +
-     read_file("README.rst").decode('latin-1')  + "\n\n" +
-     read_file("CHANGES.rst").decode('latin-1')))
+long_description = (
+    open("README.rst").read()  +
```

Comment:
Because it's just good practice to clean up after yourself?

There are tools like pyroma and others that import the setup file and then do a bunch of analysis so the script is not necessarily short lived.